build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
Add at the top the `buildscript` repositories. I would say to use `jcenter()` for the plugins...

Reply:
Lets add the optional stuff when we add our first non-included.